copier/main.py

```diff
@@ -400,6 +410,11 @@ def jinja_env(self) -> SandboxedEnvironment:
                 "Make sure to install these extensions alongside Copier itself.\n"
                 "See the docs at https://copier.readthedocs.io/en/latest/configuring/#jinja_extensions"
             )
+        # patch the `to_json` filter to support Pydantic dataclasses
+        def to_json(value, *args, **kwargs):
+            return json.dumps(value, *args, **kwargs, default=pydantic_encoder)
+
+        env.filters["to_json"] = to_json
```

Comment:
I think you can simplify it with a partial:

```suggestion
        env.filters["to_json"] = partial(env.filters["to_json"], default=pydantic_encoder)
```

Reply:
Nice, thanks!